Profiles - ZIB 2017/conceptmap-OrderStatusCodelijst-To-ActStatus.xml

```diff
@@ -0,0 +1,98 @@
+<ConceptMap xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/conceptmap.xsd">
+  <id value="OrderStatusCodelijst-To-RequestStatus"/>
+  <extension url="http://hl7.org/fhir/StructureDefinition/concept-bidirectional">
+    <valueBoolean value="false"/>
+  </extension>
+  <url value="http://nictiz.nl/fhir/ConceptMap/OrderStatusCodelijst-To-RequestStatus"/>
+  <version value="2.1.1"/>
+  <name value="OrderStatusCodelijst-To-RequestStatus"/>
+  <title value="HCIM OrderStatusCodelijst to RequestStatus"/>
+  <status value="active"/>
+  <publisher value="Nictiz"/>
+  <contact>
+    <name value="Nictiz"/>
+    <telecom>
+      <system value="email"/>
+      <value value="info@nictiz.nl"/>
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="Maps the HCIM ProcedureRequest release 2017 terminology for OrderStatus to the FHIR STU3 terminology associated with Codesystem.RequestStatus"/>
+  <purpose value="To aid in mapping from HCIM ProcedureRequest value set OrderStatus to FHIR STU3 Codesystem.RequestStatus"/>
+  <copyright value="CC0"/>
+  <sourceUri value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.16.1.2--20171231000000"/>
+  <targetUri value="http://hl7.org/fhir/ValueSet/request-status"/>
+  <group>
+    <source value="http://snomed.info/sct"/>
+    <target value="http://hl7.org/fhir/ValueSet/request-status"/>
+    <targetVersion value="http://hl7.org/fhir/ValueSet/request-status version 3"/>
```

Comment:
Ik geloof niet het CodeSystem geversioneerd is, dit kan volgens mij weg

Reply:
Ik kwam dit in een andere ConceptMap tegen, maar niet overal dus ik haal 'm dan weg.